test/addons-napi/test_general/testInstanceOf.js

```diff
@@ -34,9 +34,18 @@ function assertThrows(statement) {
 }
 
 function testFile(fileName) {
-  const contents = fs.readFileSync(fileName, { encoding: 'utf8' });
-  eval(contents.replace(/[(]([^\s(]+)\s+instanceof\s+([^)]+)[)]/g,
-                        '(addon.doInstanceOf($1, $2))'));
+  try {
+    const contents = fs.readFileSync(fileName, { encoding: 'utf8' });
+    eval(contents.replace(/[(]([^\s(]+)\s+instanceof\s+([^)]+)[)]/g,
+                          '(addon.doInstanceOf($1, $2))'));
+  } catch (err) {
+    // This test depends on V8 test files, which may not exist in downloaded
+    // archives. Emit a warning if the tests cannot be found instead of failing.
+    if (err.code === 'ENOENT')
```

Comment:
@cjihrig I propose to add `&& !fs.existsSync(path.resolve(__dirname, '../../../deps/v8/test'))`.
If the file is not found and the test directory exists, this is unexpected because the test directory should be absent from the source tarball.